public/index.html

```diff
@@ -28,7 +28,7 @@
 
       app.onInit = function () {
         this.canvas = document.getElementById("canvas");
-        this.context = this.canvas.getContext("2d");
+
```

Comment:
https://github.com/TalhaKhamoor/PingPongGame/pull/5/commits/bb4e33ee47d4df9d2fb4d2a40e445b0412815019

except the first one all of your commits are too big
this commit is unreviewable..